packages/typespec-azure-resource-manager/lib/models.tsp

```diff
@@ -169,7 +170,8 @@ enum ResourceProvisioningState {
 @doc("Contains a default provisioningState property to be spread into resource property types")
 model DefaultProvisioningStateProperty {
   @doc("The provisioning state of the resource.")
-  provisioningState: ResourceProvisioningState;
+  @visibility("read")
+  provisioningState?: ResourceProvisioningState;
```

Comment:
that's going to cause a diff in the swagger and new linting validation right?

Reply:
Yes. Though this is an older model that are not showing up in our recent samples, so usage is minimal. I have searched public main and it is not used by any spec. Looking at RPSaaSxxx now.